### PR TITLE
Add license of depot

### PR DESCRIPTION
**Pull request by dannyxx001**

### CODE REVIEW

File: ajax/libs/depot/package.json [hunk 1]
```diff
@@ -18,4 +18,5 @@
     "type": "git",
     "url": "https://github.com/mkuklis/depot.js"
   }
+  ,"license": "MIT"
```
Comment (Amomo) [inline ajax/libs/depot/package.json]: Please use [JSONLint](http://jsonlint.com/) to validate your package.json. :-)

**Comment by Amomo** [comment]
PR for #6324

**Comment by PeterDaveHello** [comment]
@Amomo Hey let then do it, i will say it later.

**Comment by PeterDaveHello** [comment]
@Amomo *them

**Comment by Amomo** [comment]
@PeterDaveHello ok, I see.

**Comment by Amomo** [comment]
Hi @dannyxx001 please try again! Try to finish a good PR.
It's welcome to ask us if you need any help! :-D

btw, depot has been picked by #6340, please pick another library for redo.
Thank you!

**Pull request closed without merging**